### PR TITLE
Make loader runnable from any path

### PR DESCRIPTION
**Pull request by jukoson**
- Make loarder and benchmark runnable from any path. The scripts will change into their respective directories
- fix python version to explicitly use 3.6. This is helpful in environments with multiple python versions, eg. in AWS we get weird behaviour when linking python3 to python 3.6 instead of 3.5

### CODE REVIEW

**Comment by sdressler** [comment]
That PR needs to go to develop, not master.

File: scripts/swarm64_run_tpc_benchmark_internal [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```
Comment (sdressler) [inline scripts/swarm64_run_tpc_benchmark_internal]: Nope, no good idea. Breaks on systems with Python 3.7/3.8. It needs a proper version check.

File: swarm64_run_tpc_benchmark [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 import sys
 
-from logging.config import fileConfig
+from os import chdir
 
+from logging.config import fileConfig
 from swarm64_tpc_toolkit.streams import Streams
 
-
+chdir(sys.path[0])
```
Comment (sdressler) [inline swarm64_run_tpc_benchmark]: That does not feel correct to me. It should then rather handle absolute paths properly.

File: swarm64_run_tpc_benchmark [hunk 3]
```diff
@@ -1,15 +1,16 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```
Comment (sdressler) [inline swarm64_run_tpc_benchmark]: Same problem with the Py version. Also, on develop, IMO, the script checks the version already.

**Review by sdressler — CHANGES_REQUESTED** [review]

File: swarm64_run_tpc_benchmark (hunk 2 above)
Comment (jukoson) [reply swarm64_run_tpc_benchmark]: Do you have a suggestion ?

Comment (sdressler) [reply swarm64_run_tpc_benchmark]: For instance methods listed in https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory to get the absolute path to the CWD and then use that for all remaining relative paths and turn them into absolute ones with `pathlib`.

**Review by sdressler — APPROVED** [review]

**Pull request merged**